documentation/test-plan.md

```diff
@@ -48,73 +48,73 @@ This test plan covers unit, integration, acceptance, regression, and load testin
 ### Unit Tests
 
 1. User Management
-   1. Return `201 Created` upon POST request to `/api/auth/register` with valid payload.
-   2. Return `400 Bad Request` upon POST request to `/api/auth/register` with an already used username.
-   3. Return `200 OK` and a JWT upon POST to `/api/auth/login` with correct credentials.
-   4. Return `400 Bad Request` upon POST to `/api/auth/login` with incorrect credentials.
-   5. Return `200 OK` upon POST to `/api/auth/reset-password` with the correct security answer.
-   6. Return `400 Bad Request` upon POST to `/api/auth/reset-password` with the incorrect security answer.
-   7. Return `400 Bad Request` upon POST to `/api/auth/reset-password` with a missing payload.
-   8. Return `200 OK` upon PUT to `/api/account` with valid new user data.
-   9. Return `400 Bad Request` upon PUT to `/api/account` with a new username that already exists.
-   10. Return `200 OK` upon DELETE to `/api/account` from authenticated session.
+   1. Return `201 Created` upon POST request to `/auth/register` with valid payload.
+   2. Return `400 Bad Request` upon POST request to `/auth/register` with an already used username.
+   3. Return `200 OK` and a JWT upon POST to `/auth/login` with correct credentials.
+   4. Return `400 Bad Request` upon POST to `/auth/login` with incorrect credentials.
+   5. Return `200 OK` upon POST to `/auth/reset-password` with the correct security answer.
+   6. Return `400 Bad Request` upon POST to `/auth/reset-password` with the incorrect security answer.
+   7. Return `400 Bad Request` upon POST to `/auth/reset-password` with a missing payload.
+   8. Return `200 OK` upon PUT to `/account` with valid new user data.
+   9. Return `400 Bad Request` upon PUT to `/account` with a new username that already exists.
+   10. Return `200 OK` upon DELETE to `/account` from authenticated session.
```

Comment:
shouldn't these be `/users` be in these as well or am I dumb (?)

Reply:
omg ty yes you're right, will fix